pom.xml

```diff
@@ -447,6 +447,12 @@
 				<type>jar</type>
 			</dependency>
 			<dependency>
+		    	<groupId>com.tdunning</groupId>
+		    	<artifactId>json</artifactId>
+				<version>1.2-SNAPSHOT</version>
```

Comment:
@bitstorm - is everything working with 1.3, now?

If yes we can integrate this branch I think.